examples/classification/helper.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from typing import Generator, Iterable, List, Optional, Tuple, Union
+
+import numpy
+
+import cv2
+from sparseml.utils import load_numpy
+from sparsezoo import Zoo
+from sparsezoo.utils import load_numpy_list
+
+
+"""
+A file with utility functions and helper classes for loading data
+"""
+
+
+class _BatchLoader:
```

Comment:
let's make sure to migrate this when this lands in sparsezoo

Reply:
will take care!